py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -408,6 +407,9 @@ def contains_metadata(gm: torch.fx.GraphModule) -> bool:
     # Generate the corresponding TRT Module for those
     for name, _ in partitioned_module.named_children():
         submodule = getattr(partitioned_module, name)
+        # filter on the GraphModule
+        if not isinstance(submodule, torch.fx.graph_module.GraphModule):
```

Comment:
What is the logic behind this?

Reply:
partitioned_module has named_children() submodule which is not torch.fx.graph_module.GraphModule type, we should only filter on the torch.fx.graph_module.GraphModule to be convertable to TRT Module.